src/main/java/com/libertymutualgroup/herman/nr/broker/clients/NewRelicSyntheticsClient.java

```diff
@@ -37,101 +37,110 @@ public void createSyntheticsMonitors(JsonNode synthetics, String policyName) {
     ObjectNode payload = ((ObjectNode)synthetics).put("name", policyName + "-synthetics");
 
     try {
-      newRelicSyntheticsRestTemplate
+      newRelicSyntheticsTemplate
           .exchange(
               "/monitors",
               HttpMethod.POST,
               new HttpEntity<>(payload, httpHeaders),
               Void.class
-              );
+          );
     } catch (Exception e) {
       throw new RuntimeException(String.format("Error Creating Synthetics Monitor: %s", payload), e);
     }
   }
 
   public void deleteExistingSyntheticsMonitors(String policyName) {
-    LOG.info("Delete any existing Synthetics Monitors for {}", policyName);
+    LOG.info("Deleting any existing Synthetics Monitors for {}", policyName);
     JsonNode syntheticsMonitorsResponse;
     try {
-      syntheticsMonitorsResponse = newRelicSyntheticsRestTemplate
+      syntheticsMonitorsResponse = newRelicSyntheticsTemplate
           .exchange(
-            "/monitors",
-            HttpMethod.GET,
-            new HttpEntity<>(httpHeaders),
-            JsonNode.class
+              "/monitors",
+              HttpMethod.GET,
+              new HttpEntity<>(httpHeaders),
+              JsonNode.class
           ).getBody();
     } catch (Exception e) {
-      throw new RuntimeException(String.format("Error deleting synthetics monitor: %s", policyName));
+      throw new RuntimeException(
+          String.format("Error deleting synthetics monitor: %s", policyName));
     }
 
     ArrayNode monitors = (ArrayNode) syntheticsMonitorsResponse.get("monitors");
-    String id = getMonitorId(policyName);
-    LOG.info("Deleting Synthetics Monitor with ID: {}", id);
-    try {
-      newRelicSyntheticsRestTemplate
-          .exchange(
-              String.format("/monitors/%s" ,id),
-              HttpMethod.DELETE,
-              new HttpEntity<>(httpHeaders),
-              Void.class
-          );
-    } catch (Exception e) {
-      throw new RuntimeException(String.format("Error deleting synthetics monitor: %s", policyName));
+    if (monitors.size() > 0) {
```

Comment:
Not a huge deal, but spacing seems off here. Looks like 2 spaces, but I'm not sure if GH is just rendering it funny.

Reply:
Yeah, it does seem off. I'll auto-format and fix this.